annotations/src/main/java/com/statful/client/annotations/Timer.java

```diff
@@ -38,4 +38,9 @@
      * @return An array of aggregations to send to Statful
      */
     Aggregation[] aggregations() default {};
+
+    /**
+     * @return An int of the sampleRate to send to Statful
+     */
+    int sampleRate() default 0;
```

Comment:
if you don't specify the value is 0 (default), and in that case the sample rate will be the one on the statful client. Why would you want the sample rate to be 0? Just don't send anything at all. We can also make the default -1 ..